source/features/locked-issue-indicator.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {LockIcon} from '@primer/octicons-react';
+import * as pageDetect from 'github-url-detection';
+
+import onetime from 'onetime';
+
+import {$, elementExists} from 'select-dom';
+
+import features from '../feature-manager.js';
+import observe from '../helpers/selector-observer.js';
+
+function addLock(element: HTMLElement): void {
+	if (!elementExists('reactions-menu', element)) {
```

Comment:
This can be moved to the observed selector, as `:not(:has(reactions-menu))`, I think